File: task2/Java/src/Task2.java
import java.io.File;
import java.io.FileNotFoundException;
import java.util.Scanner;

public class Task2 {

    public static void main(String[] args) {
        //TASK2
        if (args.length != 2) {
            System.out.println("Enter circle and dot !");
            System.exit(1);
        }

        float a, b; // Координаты круга
        float R; // Радиус

        try {
            Scanner circle = new Scanner(new File(args[0]));
            a = circle.nextFloat();
            b = circle.nextFloat();
            R = circle.nextFloat();
            circle.close();

            Scanner dot = new Scanner(new File(args[1]));
            while (dot.hasNextFloat()) {
                float x = dot.nextFloat();
                float y = dot.nextFloat();
                if((x - a)* (x - a) + (y - b)* (y - b) == R*R) System.out.println(0);
                if((x - a) * (x - a) + (y - b) * (y - b) < R * R) System.out.println(1);
                if((x - a) * (x - a) + (y - b) * (y - b) > R * R) System.out.println(2);
            }
            dot.close();
        } catch (FileNotFoundException e) {
            e.printStackTrace();
        }
    }
}

File: task2/task2.cpp
#include <iostream>
#include <string>
#include <fstream>

int main(int argc, char* argv[]) {

	//TASK2
	if (argc != 3) {
		printf("Enter circle and dot !\n");
		exit(1);
	}

	float a, b; // Координаты центра круга
	float R; // Радиус круга

	std::ifstream circle(argv[1]);
	if (circle.is_open())
	{
		circle >> a >> b;
		circle >> R;
	}
	circle.close();

	std::ifstream dot(argv[2]);
	if (dot.is_open())
	{
		float x, y;
		while (dot >> x >> y)
		{
			if((x - a)* (x - a) + (y - b)* (y - b) == R*R) std::cout << 0 << std::endl;
			if((x - a) * (x - a) + (y - b) * (y - b) < R * R) std::cout << 1 << std::endl;
			if((x - a) * (x - a) + (y - b) * (y - b) > R * R) std::cout << 2 << std::endl;
		}
	}
	dot.close();

	return 0;
}